frontend/src/hooks/queries/studylog.ts

```diff
@@ -73,6 +76,32 @@ export const useDeleteStudylogMutation = () => {
   });
 };
 
+export const useFetchTempSavedStudylog = () => {
+  const { user } = useContext(UserContext);
+  const { username } = user;
+
+  return useQuery([QUERY_KEY.tempSavedStudylog, username], () => requestGetTempSavedStudylog(), {
+    refetchOnWindowFocus: false,
+    cacheTime: 0,
+  });
+};
+
+export const useCreateTempSavedStudylog = () => {
+  const queryClient = useQueryClient();
+  const { user } = useContext(UserContext);
+  const { username } = user;
+
+  return useMutation((body: TempSavedStudyLogForm) => requestPostTempSavedStudylog(body), {
```

Comment:
하나의 익명 함수를 추가로 만들지 않아도 될 것 같습니다 ! ㅎㅎ

```suggestion
  return useMutation(requestPostTempSavedStudylog, {
```